rubrikcdm/examples_test.go

```diff
@@ -300,7 +300,48 @@ func ExampleCredentials_Bootstrap() {
 	managementSubnetMask := "255.255.255.0"
 	dnsSearchDomain := []string{"rubrikgosdk.lab"}
 	dnsNameServers := []string{"192.168.100.5", "192.168.100.6"}
-	ntpServers := []string{"192.168.100.5", "192.168.100.6"}
+	ntpServers := map[string]interface{}{}
+	ntpServers["ntpServer1"] = map[string]interface{}{}
+	ntpServers["ntpServer1"].(map[string]interface{})["IP"] = "192.168.100.5"
+	ntpServers["ntpServer2"] = map[string]interface{}{}
+	ntpServers["ntpServer2"].(map[string]interface{})["IP"] = "192.168.100.6"	enableEncryption := true // set to false for a Cloud Cluster
+	waitForCompletion := true
+
+	nodeConfig := map[string]string{}
+	nodeConfig["RVM1234567890"] = bootstrapNode
+	nodeConfig["RVM1234567891"] = "192.168.101.101"
+	nodeConfig["RVM1234567892"] = "192.168.101.102"
+	nodeConfig["RVM1234567893"] = "192.168.101.103"
+
+	bootstrap, err := rubrik.Bootstrap(clusterName, adminEmail, adminPassword, managementGateway, managementSubnetMask, dnsSearchDomain, dnsNameServers, ntpServers, nodeConfig, enableEncryption, waitForCompletion)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func ExampleCredentials_Bootstrap_with_Secure_NTP() {
+
+	bootstrapNode := "192.168.101.100"
+	rubrik := rubrikcdm.Connect(bootstrapNode, "", "")
+
+	clusterName := "Go-SDK"
+	adminEmail := "gosdk@rubrikgosdk.lab"
+	adminPassword := "RubrikGoSDK"
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High <!-- 0603a19e-ece2-4261-87ca-ff549d2f2b07 -->